instrumentation/rdkafka/opentelemetry-instrumentation-rdkafka.gemspec

```
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/rdkafka/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-rdkafka'
+  spec.version     = OpenTelemetry::Instrumentation::Rdkafka::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Rdkafka instrumentation for the OpenTelemetry framework'
+  spec.description = 'Rdkafka instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'opentelemetry-api', '~> 1.0.0'
+  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.18.2'
+ spec.add_dependency 'opentelemetry-common', '~> 0.19.2'
```

CI Failing Due to, otherwise i think we're good to go
```suggestion
  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.19.0'
  spec.add_dependency 'opentelemetry-common', '~> 0.19.3'
```

---

(these got bumped by release today 😄 )